content/coverpage.tex

```diff
@@ -60,6 +60,7 @@ \subsection*{\openshmem 1.6}
 \item Aurelien Bouteiller, \ac{UTK}
 \item Mark Brown, Intel
 \item Bob Cernohous, \ac{HPE}
+\item Matthew Davis, Georgia Tech
```

Comment:
@ronawho, could you please add `Youssef Elmougy, Georgia Tech` (@youssefelmougy)?

Reply:
@jdinan can you add contributors as a doc edit or do I need to add them? If I need to add, is that something that goes in the frontmatter RC2 or a new PR?

Reply:
@ahayashi Thank you! I'll include this change as a doc. edit before we publish the final draft.